src/active_object.rs

```diff
@@ -1,10 +1,12 @@
 // Implements http://rosettacode.org/wiki/Active_object
-#![allow(unstable)]
+#![feature(std_misc)]
+#![feature(io)]
+#![feature(core)]
```

Comment:
it is possible indeed. The reason I did it this way is that all these attributes should be temporary and disappear one by one when things get stabilized. When that happens it's much easier to replace/remove the whole line in multiple files.